backend/server.js

```diff
@@ -84,6 +84,46 @@ app.post(
     },
 );
 
+// FILE server.js
+
+// Route for getting json containing all relevant user data
+app.get('/users/:userId', function(req, res) {
+  console.log('test');
+  usersModel.findOne({userId: req.params.userId}, function(err, userData) {
+    if (err) {
+      res.send(err);
+    } else {
+      const relevantData = {
+        email: userData.email,
+        username: userData.username,
+      };
+      res.send(relevantData);
+    }
+  });
+});
+
+// Route for getting username
+app.get('/users/:userId/username', function(req, res) {
```

Comment:
I like having multiple options for retrieving user data, if only certain fields are needed